packages/tasks/src/pipelines.ts

```diff
@@ -635,6 +635,16 @@ export const PIPELINE_DATA = {
 		modality: "cv",
 		color: "yellow",
 	},
+	"text-to-3d": {
```

Comment:
We should start tagging some repos with this, otherwise https://huggingface.co/datasets?other=text-to-3d will be empty. You can add `hideInDatasets: false` here and it will not show the tag at left of hf.co/datasets, but I think we do have 3D datasets to add this tag.